integration/qbft/scenarios/integration_test_type.go

```diff
@@ -227,55 +209,67 @@ func (it *IntegrationTest) Run() error {
 
 	<-time.After(lastDutyTime + (4 * time.Second))
 
-	for operatorID, expectedErr := range it.StartDutyErrors {
-		if actualErr, ok := actualErrMap.Load(operatorID); !ok {
-			if expectedErr != nil {
-				return fmt.Errorf("expected an error")
-			}
-		} else {
-			aerr, ok := actualErr.(error)
-			if !ok && expectedErr != nil {
-				return fmt.Errorf("expected an error")
-			}
-			if !errors.Is(aerr, expectedErr) {
-				return fmt.Errorf("got error different from expected (expected %v): %w", expectedErr, actualErr.(error))
-			}
+	err = nil
+	validateErr := func(k, v any) bool {
```

Comment:
I think we need to rename `v` to something like `actualErr`, probably `k` as well